pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <artifactId>openam-auth-trees</artifactId>
         <groupId>org.forgerock.am</groupId>
-        <version>6.5.0</version>
+        <version>7.2.0</version>
```

Comment:
We shouldn't really extend AM artifacts, instead this Maven module should use the auth trees BOM, like: https://github.com/ForgeRock/get-profile-property-auth-tree-node/blob/master/pom.xml

Reply:
removed parent and updated dependencies